artifactory-enterprise/azuredeploy.parameters.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
```

Comment:
remove parameters that have defaultValues

Reply:
done